query/src/sql/exec/data_schema_builder.rs

```diff
@@ -34,8 +35,10 @@ impl<'a> DataSchemaBuilder<'a> {
         DataSchemaBuilder { metadata }
     }
 
+    #[allow(dead_code)]
```

Comment:
Seems `#[allow(dead_code)]` in the file and `_aggregate: &AggregatePlan,` can be deleted.

Reply:
Fixed.